sklearn/model_selection/_validation.py

```diff
@@ -464,6 +466,13 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
         score_time = 0.0
         if error_score == 'raise':
             raise
+        elif error_score == 'raise-deprecating':
+            warnings.warn("Estimator fit failed. In the future,"
+                          "raise-deprecating error will not be"
```

Comment:
No need to reference raise-deprecating, just explain how the default error_score behaviour will change